Prace_domowe/Praca_domowa2/Gruszkowska_Klaudia/PD2_Klaudia_Gruszkowska.ipynb

```diff
@@ -0,0 +1,1800 @@
+{
```

Comment:
<p>Oryginalny pomysł na wykres, ale z uwagi na to, że kolejne iteracje są niezależne i nie tworzą szeregu czasowego nie powinno się używać wykresu liniowego. Wówczas lepiej byłoby punktowy. Niemniej jednak z uwagi na to, że nie jest to szereg czasowy, lepiej sprawdziłyby się dwa boxploty - one jeszcze lepiej pokażą rozrzut wartości. Niemniej jednak fakt, że rozrzut policzono poniżej jest "okolicznością łagodzącą". Jest to uwaga na przyszłość - trzeba uważać z niektórymi typami wykresów.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>